groups/prometheus/module-ec2-instances/variables.tf

```diff
@@ -0,0 +1,48 @@
+# Instance
+variable "instance_type" {
+  type = string
+}
+variable "instance_count" {
+  type = number
+}
+variable "ami_version_pattern" {
+  type = string
+}
+
+# Networking
+variable "application_subnets" {
+  type = list(string)
+}
+variable "vpc_security_group_ids" {
+  type = string
+}
+
+# DNS
+variable "zone_name" {
+  type = string
+}
+
+# SSH
+variable "ssh_keyname" {
+  type = string
+}
+variable "private_key_path" {
+  type = string
+}
+
+# Tags
+variable "tag_environment" {
+  type = string
+}
+variable "tag_service" {
+  type = string
+}
+
+# Concourse Prometheus
+variable "web_fqdn" {
```

Comment:
You may want to consider renaming service-specific variables to highlight the use case, particularly if we're ordering variables in this file alphabetically as it leads to a natural grouping of things. Names like `prometheus_web_fqdn` and `prometheus_metrics_port` might work better.